### ISSUE TITLE
changing WiFi Channel

### ISSUE THREAD

**Issue by Spite1** [martin-ger/esp_wifi_repeater#422]
Hi,

is there a way to change the WiFi Channel of the repeater

**Comment by martin-ger** [comment]
No, the channel of the AP must be the same as the client STA the uplink AP. This is a technical restriction of the SATTIONAP mode.

**Comment by Spite1** [comment]
thank you very much for your answer

**Comment by Cloolalang** [comment]
It would be pretty useful to set the WIFI channels when using the ENC28J60 Ethernet Support for a cheap AP
